config/network.go

```diff
@@ -83,7 +107,21 @@ func (n *NetworkConfig) Validate() error {
 			// we don't need to validate RPC endpoints or private keys for simulated networks
 			continue
```

Comment:
don't we want to validate `forkConfigs` for simulated networks?

Reply:
Do we want to fork simulated? Can you elaborate one use cae for that?

Reply:
For fork config set up please refer to the added tests

Reply:
rather wondering about a case when we are forking Sepolia, how will that look? we will use `SEPOLIA` as `selected_networks` and just modify URLs so that they point to simulated env instead of live Sepolia RPC?

Reply:
Here's the example it's there in the test `TestVariousNetworkConfig` -
```
selected_networks = ["KROMA_SEPOLIA"]
[fork_config.KROMA_SEPOLIA]
url = "ws://localhost:8546"
block_number = 100
```

Reply:
that doesn't look right. `url` is the url from which we fork it, why would we for it from local node?